assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
@Ap0c Yeap, we should extract the window from the user reducer. I created a [card](https://trello.com/c/Z3Mjst6V/714-refactor-extract-the-window-object-from-the-userreducer-and-add-jest-test) to address this refactor.